test/gpu/interop/cuBLAS.skipif

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+from distutils.spawn import find_executable
+import os
+
+missing_nvcc = find_executable("nvcc") is None
+
+# We don't have a great way to autodetect if we're going to launch onto a
+# compute node with GPUs, so we currently use `CHPL_TEST_GPU` as a proxy
+will_run_on_gpu = os.getenv('CHPL_TEST_GPU') == 'true'
```

Comment:
A nit, but for such environment variables, I prefer if we check they are defined/undefined rather than checking whether they're defined specifically as `true`.

I can adapt, but I can also imagine myself doing `export CHPL_TEST_GPU=1` or equivalent and getting bit by it or something.

Reply:
I went with this because I thought it'd be confusing if `CHPL_TEST_GPU=false` still ran the tests. I also sometimes do the `=1` thing, so probably what I really want is an "isTruthy" sort of check, but I didn't want to get into that here.

Reply:
Yeah, isTruthy would be the best. I personally find `CHPL_TEST_GPU=false` to be awkward, but I understand what you are saying. I have no problem with this as-is.